SpecEasy.Specs/BeforeEachAndAfterEachExampleSpecs/GetSetInBeforeEachExampleSpec.cs

```diff
@@ -18,16 +18,16 @@ public void Run()
             When("using a dependency", () => result = SUT.UseDependency());
 
             Given("dependencies were set up in BeforeEachExample").Verify(() =>
-                Then("it should get a value from the first dependency that was set up", () => result.ShouldEqual(123)).
-                Then("it should get a value from the second dependency that was set up", () => SUT.UseAnotherDependency().ShouldEqual(789)));
+                Then("it should get a value from the first dependency that was set up", () => result.Should().Be(123)).
+                Then("it should get a value from the second dependency that was set up", () => SUT.UseAnotherDependency().Should().Be(789)));
 
             Given("dependency overridden in a Given", () =>
             {
-                Get<IDependency>().Stub(dep => dep.GetValue()).Return(456).Repeat.Any(); // Repeat.Any replaces the previous Stub
+                Get<IDependency>().GetValue().Returns(456); // TODO: if I understand the documentation correctly, NSubstitute always replaces the previous Stub
```

Comment:
Remove/resolve TODO

Reply:
Whoops, good catch! Done. I confirmed that behavior and have removed the note.